exercises/practice/accumulate/Get-Accumulation.tests.ps1

```diff
@@ -0,0 +1,23 @@
+BeforeAll {
+    . "./Get-Accumulation.ps1"
+}
+
+Describe 'Test Get-Accumulation.ps1' {
```

Comment:
These tests look to be a subset of the tests in the canonical data of this exercise: https://github.com/exercism/problem-specifications/blob/main/exercises/accumulate/canonical-data.json Was there a specific reason for that?